heroku/README.md

```diff
@@ -9,7 +9,7 @@ This is a sample client for Facebook's [Webhooks](https://developers.facebook.co
 1. Refer to Facebook's [Webhooks sample app documentation](https://developers.facebook.com/docs/graph-api/webhooks/sample-apps) to see how to use this app.
 1. Deploy the sample app on Heroku with this button:
 
-    [![Deploy](https://www.herokucdn.com/deploy/button.svg)](https://heroku.com/deploy?template=https://github.com/fbsamples/graph-api-webhooks-samples/)
+    [![Deploy](https://www.herokucdn.com/deploy/button.svg)](https://heroku.com/deploy?template=https://github.com/fbsamples/graph-api-webhooks-samples)
```

Comment:
```suggestion
    [![Deploy](https://www.herokucdn.com/deploy/button.svg)](https://heroku.com/deploy?template=https://www.github.com/fbsamples/graph-api-webhooks-samples)
```

Reply:
Removing a trailing slash at the end of the URL does not accomplish what your PR sets out to do.